Terraform/ec2-with-eni/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+# .tfvars Sample File
+
+
+# Region Creds
+symphony_ip = "<symphony_ip>"
+# access_key = "<access_key>"
+# secret_key = "<secret_key>"
+
+credentials_file = "<path_to_creds_file>" # example:"/$HOME/.aws/credentials"
```

Comment:
should be explained how to use

Reply:
I have explained in the main README file of all terraform examples